server/resourcemanager/testutil/operations.go

```diff
@@ -0,0 +1,115 @@
+package testutil
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	defaultOperations = []operationTester{
```

Comment:
these are the operations intended for the "general public" resources implementation